reporter/src/main/kotlin/reporters/WebAppReporter.kt

```diff
@@ -34,17 +35,17 @@ class WebAppReporter : Reporter {
         input: ReporterInput
     ) {
         val template = javaClass.classLoader.getResource("scan-report-template.html").readText()
-        val resultJson = jsonMapper.writeValueAsString(input.ortResult)
+        val evaluatedModel = EvaluatedModel.create(input)
 
-        val relevantResolutions = input.resolutionProvider.getResolutionsFor(input.ortResult)
-        val resolutionsJson = jsonMapper.writeValueAsString(relevantResolutions)
-
-        val result = template
-            .replace("id=\"ort-report-data\"><", "id=\"ort-report-data\">$resultJson<")
-            .replace("id=\"ort-report-resolution-data\"><", "id=\"ort-report-resolution-data\">$resolutionsJson<")
+        val prefix = template.substringBefore("id=\"ort-report-data\"><")
+        val suffix = template.substringAfter("id=\"ort-report-data\"><")
 
         outputStream.bufferedWriter().use {
-            it.write(result)
+            it.write(prefix)
+            it.write("id=\"ort-report-data\">")
```

Comment:
Could combine this with the above to `it.write("${prefix}id=\"ort-report-data\">")`.